### PR TITLE
Add changelog entries from #4622

### PR DESCRIPTION
**Pull request by jasongrout**
Add changelog entries from #4622

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,106 @@
...
+* Main area widgets like the notebook, editor, console, and launcher now inherit from a common `MainAreaWidget` class. The `MainAreaWidget` class provides a consistent interface with a content area (`.content`) and a toolbar (`.toolbar`), and consistent focus handling and activation. Extension authors may consider inheriting from the `MainAreaWidget` for consistency. Several changes this caused include:
+
+  * The notebook panel `.notebook` attribute is renamed to `.content`.
+  * The text editor widget's editor is `.content.editor` rather than just `.editor`.
+  * Document widgets implement `IDocumentWidget<ContentType>`, instead of using `DocumentRegistry.IReadyWidget`. A `DocumentWidget` has the `.toolbar` and `.content` attributes from `MainAreaWidget` and adds a context attribute.
+  * Mime documents use a `MimeContent` widget embedded inside of a `DocumentWidget` now.
+
+  ([#3499](https://github.com/jupyterlab/jupyterlab/pull/3499), [4453](https://github.com/jupyterlab/jupyterlab/pull/4453))
+
+* The notebook panel `.notebook` attribute has been renamed to `.content` for consistency with other main area widgets.
```
Comment (ian-r-rose) [inline CHANGELOG.md]: This is duplicated.

**Comment by jasongrout** [comment]
@ian-r-rose - better?

I think it's good to go now.

**Comment by jasongrout** [comment]
ping also @jzf2101 - I think I'm done with the content now, and I did a pass for consistency with sentence structure.

**Comment by jasongrout** [comment]
Travis failure is from the docs link that will work when we release...

**Pull request merged**